apps/desktop/src/lib/routes/routes.svelte.ts

```diff
@@ -48,3 +48,7 @@ export class DesktopRoutesService {
 		return `/${targetProjectId}/${targetRestPath}`;
 	}
 }
+
+export function stackPath(projectId: string, stackId: string) {
+	return `/${projectId}/workspace/${stackId}`;
+}
```

Comment:
@ndom91 I'm wondering if the class above is needed? There seems to be different requirements for web routes, but here the reactivity doesn't benefit from the class.. and I'm thinking this is unlikely something we'll want to mock in testing.

Reply:
🤔

I'd be for making a routing class exactly like what is in the web, it seems like a very helpful way of handling things like sidebar state and breadcrumbs.

Surly we should look to finding a way of making our tests work with .svelte.ts files, rather than trying to avoid them?

Reply:
My point is, the class is currently just decorative, it appears to me it serves no purpose other than making these functions less accessible.